ocs_authentication/auth_profile/migrations/0001_initial.py

```diff
@@ -1,6 +1,5 @@
-# Generated by Django 3.1.11 on 2021-10-19 20:39
+# Generated by Django 4.0 on 2021-12-15 02:03
```

Comment:
Does it matter that these versions changed? Isn't everything else Django 3? Maybe this project's setup.py should also be constrained similarly?

Reply:
yeah thanks for pointing that out. I locked down the version and regenerated the migration.